components/scream/src/physics/share/physics_test_data.hpp

```diff
@@ -255,7 +256,51 @@ class PhysicsTestData
   // the member to a range.
   // Example, to use a -1 to 1 range for wthl member:
   // d.randomize({ {d.wthl, {-1, 1}} });
-  void randomize(const std::vector<std::pair<void*, std::pair<Real, Real> > >& ranges = {});
+  template <typename Engine>
+  void randomize(Engine& engine, const std::vector<std::pair<void*, std::pair<Real, Real> > >& ranges = {})
```

Comment:
Basically the same as the original code but needed to be moved to header in order to take a template Engine argument.